api/py/test/sample/joins/sample_team/sample_deprecation_join.py

```diff
@@ -0,0 +1,28 @@
+#     Copyright (C) 2023 The Chronon Authors.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from ai.chronon.join import Join, JoinPart
+from group_bys.sample_team import sample_deprecation_group_by
+from sources import test_sources
+
+v1 = Join(
+    left=test_sources.staging_entities,
+    right_parts=[JoinPart(group_by=sample_deprecation_group_by.v1, tags={"experimental": True})],
+    table_properties={"config_json": """{"sample_key": "sample_value"}"""},
+    output_namespace="sample_namespace",
```

Comment:
Suggest keeping example simple by removing attributes not required for test like `env, tags` etc

Reply:
Updated